### PR TITLE
[bump] package version for server

### PR DESCRIPTION
**Pull request by ksbrar**
@fluidframework/build-common:     0.20.0 (unchanged)
     @fluidframework/eslint-config-fluid:     0.20.1 (unchanged)
      @fluidframework/common-definitions:     0.20.0 (unchanged)
            @fluidframework/common-utils:     0.25.1 (unchanged)
                                  Server:   0.1014.0 -> 0.1014.1
                                  Client:     0.28.0 (unchanged)
                         generator-fluid:      0.3.0 (unchanged)
                             tinylicious:      0.2.0 (unchanged)
                             dice-roller:      0.0.1 (unchanged)

Also remove pre-release dependencies for server
                                  Server -> ^0.1014.0

### CODE REVIEW

File: server/tinylicious/package.json [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
-    "@fluidframework/protocol-definitions": "^0.1014.0-0",
-    "@fluidframework/server-lambdas": "^0.1014.0-0",
-    "@fluidframework/server-local-server": "^0.1014.0-0",
-    "@fluidframework/server-memory-orderer": "^0.1014.0-0",
-    "@fluidframework/server-services-client": "^0.1014.0-0",
-    "@fluidframework/server-services-core": "^0.1014.0-0",
-    "@fluidframework/server-services-shared": "^0.1014.0-0",
-    "@fluidframework/server-services-utils": "^0.1014.0-0",
-    "@fluidframework/server-test-utils": "^0.1014.0-0",
+    "@fluidframework/gitresources": "^0.1014.0",
```
Comment (vladsud) [inline server/tinylicious/package.json:28]: Is this right? Shouldn't it be ^0.1014.1?

File: packages/drivers/local-driver/package.json [hunk 2]
```diff
@@ -56,12 +56,12 @@
...
     "@fluidframework/driver-definitions": "^0.28.0",
     "@fluidframework/driver-utils": "^0.28.0",
-    "@fluidframework/protocol-definitions": "^0.1014.0-0",
+    "@fluidframework/protocol-definitions": "^0.1014.0",
     "@fluidframework/routerlicious-driver": "^0.28.0",
-    "@fluidframework/server-local-server": "^0.1014.0-0",
-    "@fluidframework/server-services-client": "^0.1014.0-0",
-    "@fluidframework/server-services-core": "^0.1014.0-0",
-    "@fluidframework/server-test-utils": "^0.1014.0-0",
+    "@fluidframework/server-local-server": "^0.1014.0",
```
Comment (vladsud) [inline packages/drivers/local-driver/package.json:61]: Same here - shouldn't it be ^0.1014.1?

**Review by vladsud — APPROVED** [review]

File: server/tinylicious/package.json (hunk 1 above)
Comment (curtisman) [reply server/tinylicious/package.json:28]: We depends on the prerelease version before. 
We just released 0.1014.0, so we are just removing the pre-release suffix

**Pull request merged**